content/en/blog/2023/notary-completes-fuzzing-security-audit.md

```diff
@@ -0,0 +1,42 @@
+---
+title: The Notary Project completes fuzzing security audit
+author:  "Adam Korczynski, Feynman Zhou"
+date: 2023-03-17
+draft: false
+---
+
+> Reviewed by Pritesh Bandi, Samir Kakkar, Shiwei Zhang, Toddy Mladenov, Vani Rao, Yi Zha
+
+The Notary Project is happy to announce the completion of the fuzzing audit of Notation libraries. The audit was carried out by Ada Logics and is part of [an initiative by the CNCF](https://www.cncf.io/blog/2022/06/28/improving-security-by-fuzzing-the-cncf-landscape/) to bring fuzzing to the CNCF landscape. The audit spanned several months in late 2022 and early 2023 and resulted in 20 fuzzers written for 3 Notary sub-projects and 2 issues found, one of which was security-critical. 
+
+The full report from the audit is available [here](https://github.com/notaryproject/notaryproject/tree/main/security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf). 
+
+## About the Notary Project
+
+The Notary Project is an open standard and tooling for signing and verifying artifacts and safeguarding the software delivery security from development to deployment and ensuring integrity. It was started at Docker in 2015 and powers [Docker content trust](https://docs.docker.com/engine/security/trust/) which is the docker trust set of commands. With the Notary Project, users can assign trust to data and verify the integrity of the signed data. Notary is an implementation of The Update Framework (TUF) and is used in many enterprise-grade cloud solutions.
+
+The Notary Project was accepted into the CNCF in October 2017 and is hosted as an incubating project. Contributors are from organizations including Microsoft, AWS, Docker, and independent individuals. The fuzzing audit was limited to the following sub-projects:
+
+- [Notary](https://github.com/notaryproject/notary): A server and a client for running and interacting with trusted collections. 
+- [Notation-go](https://github.com/notaryproject/notation-go): a collection of libraries for supporting signing, verifying OCI artifacts. Based on the Notary standard. 
+- [Notation-core-go](https://github.com/notaryproject/notation-core-go): Crypto library for signature envelope, and signature format specific implementation.
+
+Notation-go and Notation-core are sub-projects for the Notary project which is an effort to build a signing framework to be used in every container image registry, and where signatures are easily distributed with images, [which are features that were missing in Notary](https://www.docker.com/blog/notary-v2-project-update/). 
```

Comment:
I believe this line should be moved up ,before #18. Essentially we need to introduce  "Notation-go" and "Notation-core-go" as the recent work of the Notary project before we mention that Fuzzing test was done on these

Reply:
Good catch. I have moved it to line 18.